tests/integration/test_s3_table_function_with_environment_proxy/test.py

```diff
@@ -0,0 +1,61 @@
+import logging
+import time
+
+import pytest
+from helpers.cluster import ClickHouseCluster
+
+
+@pytest.fixture(scope="module")
+def cluster():
+    try:
+        cluster = ClickHouseCluster(__file__)
+
+        cluster.add_instance(
+            "node1",
+            with_minio=True,
+            env_variables={
+                "http_proxy": "http://proxy1",
+                "https_proxy": "http://proxy1",
```

Comment:
Please add a note here that this is a deliberate choice of configuration, and we are basically routing HTTPS requests via HTTP proxy because <insert reason here>`
Or use "https://proxy1" here

Reply:
Used the `https` one.